### ISSUE TITLE
Add libwebsocket

### ISSUE THREAD

**Issue by njam** [cargomedia/debian-packages#23]
We've been using HEAD, but doesn't work anymore. @landswellsong suggests to use `v1.5-chrome47-firefox41`:
See https://github.com/cargomedia/janus-gateway-rtpbroadcast/pull/37/files
And http://git.libwebsockets.org/cgi-bin/cgit/libwebsockets/tree/changelog

@kris-lab @ppp0 @dergraf fyi

**Comment by landswellsong** [comment]
@njam my 5 ¢: it does work, but the `configure` of `janus` wants the `libwebsocket_get_internal_extensions` function inside `libwebsockets.so` and in later code this function is re-defined as a macro instead. So the code would compile but configure phase fails. This might be fixed in newer `janus` code too, I've not checked since I keep the revision we sticked to when we started. I took the most recent tag that does not have this change as a reference.
